src/test/java/bio/terra/service/resourcemanagement/google/ResourceServiceUnitTest.java

```diff
@@ -16,30 +20,29 @@
 import bio.terra.service.dataset.DatasetStorageAccountDao;
 import bio.terra.service.dataset.DatasetSummary;
 import bio.terra.service.dataset.GoogleStorageResource;
+import bio.terra.service.profile.ProfileDao;
 import bio.terra.service.resourcemanagement.AzureDataLocationSelector;
 import bio.terra.service.resourcemanagement.ResourceService;
 import bio.terra.service.resourcemanagement.azure.AzureApplicationDeploymentResource;
 import bio.terra.service.resourcemanagement.azure.AzureApplicationDeploymentService;
+import bio.terra.service.resourcemanagement.azure.AzureContainerPdao;
 import bio.terra.service.resourcemanagement.azure.AzureStorageAccountResource;
 import bio.terra.service.resourcemanagement.azure.AzureStorageAccountService;
+import bio.terra.service.snapshot.SnapshotStorageAccountDao;
 import java.util.List;
 import java.util.UUID;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
-import org.mockito.InjectMocks;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
-import org.mockito.MockitoAnnotations;
-import org.mockito.junit.MockitoJUnitRunner;
-import org.springframework.test.context.ActiveProfiles;
+import org.mockito.junit.jupiter.MockitoExtension;
 
-@ActiveProfiles({"google", "unittest"})
-@Category(Unit.class)
-@RunWith(MockitoJUnitRunner.StrictStubs.class)
-public class ResourceServiceUnitTest {
+@ExtendWith(MockitoExtension.class)
+@Tag(Unit.TAG)
+class ResourceServiceUnitTest {
 
-  @InjectMocks private ResourceService resourceService;
```

Comment:
Yes, this effectively calls the object's constructor with mocks from this class. If it can't find a mock, it passes `null`. Since this supports field injection, it's great for testing classes where constructor injection isn't used. If we wanted to migrate away from constructor injection we'd have to use this or something similar in our tests.

This annotation should be equivalent to the `new` call below, although in some cases I used `mock()` instead of passing in `null`.

In some ways using `InjectMocks` is an improvement, although it does make it less obvious to the reader how the `@Mock` objects are being used by the object being tested. But since we don't use it commonly in our code I thought it would be better to rewrite without it for now.